src/monitor/commands/accept-schedule.command.ts

```diff
@@ -37,50 +41,71 @@ export class AcceptScheduleCommand {
       schedule.end,
     );
 
-    await this.prismaService.scheduleMonitoring.update({
+    await this.prisma.scheduleMonitoring.update({
       data: { id_status: ScheduleStatus.CONFIRMED },
       where: { id: schedule.id },
     });
+
+    const user = await this.prisma.user.findUnique({
+      where: { id: userId },
+    });
+
+    const email = schedule.student.contact_email;
+    const sub = 'A ajuda está vindo!';
+    const template = 'accept_scheduled_monitoring';
+    const context = {
+      status: 'Confirmado',
+      name: user.name,
+      date: schedule.start.toLocaleDateString('pt-BR'),
+      start: schedule.start.toLocaleTimeString('pt-BR').slice(0, 5),
+      end: schedule.end.toLocaleTimeString('pt-BR').slice(0, 5),
+    };
+
+    this.emailService.sendEmailAcceptScheduledMonitoring(
```

Comment:
O que acha desse nome, já que é algo que já aconteceu?

```suggestion
    this.emailService.sendAcceptedScheduleEmail(
```